services/order.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.utils.timezone import make_aware
+
+from db.models import User
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(tickets: list[dict], username: str, date: str = None) -> Order:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
The `create_order` function should handle the case where a `User` with the given username does not exist. Consider using `get_object_or_404` or handling the `DoesNotExist` exception to provide a more user-friendly error message.